test/performance/core/simd/simd_iota_benchmark.cpp

```diff
@@ -0,0 +1,185 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <seqan3/std/ranges>
+#include <vector>
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/core/simd/simd_algorithm.hpp>
+#include <seqan3/core/simd/simd.hpp>
+#include <seqan3/core/simd/view_iota_simd.hpp>
+
+// ============================================================================
+//  simd_iota_iterator
+// ============================================================================
+
+template <typename simd_t>
+struct simd_iota_iterator_function
```

Comment:
```suggestion
struct iota_simd_view
```

Maybe you want to suffix all of these functors with `_benchmark` to make it more clear that these are the benchmarks